server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -0,0 +1,40 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
+
+import org.slf4j.Logger;
+
+public class CloudFoundryLifecycle extends AbstractLifecycle {
+
+    private final TransformationContext context;
+    private final Logger logger = super.logger;
```

Comment:
Why do you define a field that has been already defined?

This line is useless. you can always access the logger using `logger` or `this.logger`

Reply:
Actually, the logger of a superclass should not be reused. Reason: The logger has the wrong context (in that case, context is `AbstractLifeCycle.class`, but should be `CloudFoundryLifecycle.class`)

Reply:
No its not in this case.

The logger gets created using the `getClass()` method wich always is exectued on the class of which the object is a instance of. Therefore this logger is in the CloudFounrdy context